ietf/meeting/management/commands/import_iesg_minutes.py

```diff
@@ -0,0 +1,301 @@
+# Copyright The IETF Trust 2023, All Rights Reserved
+
+from collections import namedtuple
+import datetime
+import os
+import re
+import shutil
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from pathlib import Path
+from zoneinfo import ZoneInfo
+from ietf.doc.models import DocEvent, Document
+
+from ietf.meeting.models import (
+    Meeting,
+    SchedTimeSessAssignment,
+    Schedule,
+    SchedulingEvent,
+    Session,
+    TimeSlot,
+)
+from ietf.name.models import DocTypeName
+
+
+def add_time_of_day(bare_datetime):
+    """Add a time for the iesg meeting based on a date and make it tzaware
+
+    From the secretariat - the telechats happened at these times:
+    2015-04-09 to present: 0700 PT America/Los Angeles
+    1993-02-01 to 2015-03-12: 1130 ET America/New York
+    1991-07-30 to 1993-01-25: 1200 ET America/New York
+    """
+    dt = None
+    if bare_datetime.year > 2015:
+        dt = bare_datetime.replace(hour=7).astimezone(ZoneInfo("America/Los_Angeles"))
+    elif bare_datetime.year == 2015:
+        if bare_datetime.month >= 4:
+            dt = bare_datetime.replace(hour=7).astimezone(
+                ZoneInfo("America/Los_Angeles")
+            )
+        else:
+            dt = bare_datetime.replace(hour=11, minute=30).astimezone(
+                ZoneInfo("America/New_York")
+            )
+    elif bare_datetime.year > 1993:
+        dt = bare_datetime.replace(hour=11, minute=30).astimezone(
+            ZoneInfo("America/New_York")
+        )
+    elif bare_datetime.year == 1993:
+        if bare_datetime.month >= 2:
+            dt = bare_datetime.replace(hour=11, minute=30).astimezone(
+                ZoneInfo("America/New_York")
+            )
+        else:
+            dt = bare_datetime.replace(hour=12).astimezone(ZoneInfo("America/New_York"))
+    else:
+        dt = bare_datetime.replace(hour=12).astimezone(ZoneInfo("America/New_York"))
+
+    return dt.astimezone(datetime.timezone.utc)
+
+
+def build_bof_coord_data():
+    CoordTuple = namedtuple("CoordTuple", "meeting_number source_name")
+
+    def utc_from_la_time(time):
```

Comment:
This helper seems fragile to me, but I may not understand what it's trying to do. What I _think_ it's meant to do is to interpret a naive `datetime` as `Los_Angeles` time and convert it to an aware `datetime` in UTC.

If that's correct, it should be `time.replace(tzinfo=Zoneinfo("America/Los_Angeles").astimezone(datetime.timezone.utc)`. As is, it _will_ do the right thing on production, but not on machines with system time zones other than US Pacific time. 

E.g., on my laptop (which uses Atlantic time),
```python
>>> naive = datetime.datetime(2016, 6, 10, 7, 0)
>>> aware = naive.astimezone(ZoneInfo("America/Los_Angeles"))
>>> str(naive)
'2016-06-10 07:00:00'
>>> str(aware)
'2016-06-10 03:00:00-07:00'
>>> str(aware.astimezone(datetime.timezone.utc))
'2016-06-10 10:00:00+00:00'
```
(note the second output - the time it's representing is 3AM in the pacific time zone, not 7AM)

On IETFA, the output is different for the same inputs:
```python
>>> str(naive)
'2016-06-10 07:00:00'
>>> str(aware)
'2016-06-10 07:00:00-07:00'
>>> str(aware.astimezone(datetime.timezone.utc))
'2016-06-10 14:00:00+00:00'
```

Using `replace()` instead, on my machine:
```python
>>> better_aware = naive.replace(tzinfo=ZoneInfo("America/Los_Angeles"))
>>> str(better_aware)
'2016-06-10 07:00:00-07:00'
>>> str(better_aware.astimezone(datetime.timezone.utc))
'2016-06-10 14:00:00+00:00'
```
and on IETFA
```python
>>> better_aware = naive.replace(tzinfo=ZoneInfo("America/Los_Angeles"))
>>> str(better_aware)
'2016-06-10 07:00:00-07:00'
>>> str(aware.astimezone(datetime.timezone.utc))
'2016-06-10 14:00:00+00:00'
```